apis/Google.Cloud.Firestore/Google.Cloud.Firestore.IntegrationTests/AggregateQueryTest.cs

```diff
@@ -109,4 +109,14 @@ public async Task MultipleAggregations()
         Assert.Equal(Student.Data.Average(c => c.MathScore), snapshot.GetValue<double>("avg_score"));
     }
 
+    [Fact]
+    public async Task NestedAggregations()
+    {
+        CollectionReference collection = _fixture.StudentCollection;
+        var snapshot = await collection.Aggregate(AggregateField.Sum("Nested.Math"), Average("Nested.Math", "avg_score"), Count()).GetSnapshotAsync();
```

Comment:
I'd also like to check that the alias "Sum_Nested.Math" works, which is the one we would generate by default, right? I mean, I'm not certain if there are restrictions on how the alias can look like (apart from length). I tried to find it but couldn't.

Reply:
I went to put that in, and we've already got that :)

```csharp
Assert.Equal(Student.Data.Sum(c => c.Nested.Math), snapshot.GetValue<double>("Sum_Nested.Math"));
```